docs/content/overview.md

```diff
@@ -94,13 +96,26 @@ systems they must support a common hash type.
 
 ### ModTime ###
 
-The cloud storage system supports setting modification times on
-objects.  If it does then this enables a using the modification times
-as part of the sync.  If not then only the size will be checked by
-default, though the MD5SUM can be checked with the `--checksum` flag.
-
-All cloud storage systems support some kind of date on the object and
-these will be set when transferring from the cloud storage system.
+Allmost all cloud storage systems support storing modification times
+on objects, at least some kind of timestamp that are set on upload.
+If it does then this enables using them as part of the sync.
+If not, only the size will be checked by default, though the
+file hash can be checked with the `--checksum` flag.
+
+Storage systems with an `R`, for read-only, in the ModTime colum,
+means the it keeps modification times on objects, and updates them when
+uploading objects, but it does not support changing only the
+modification time (`SetModTime` operation) without re-uploading,
+possibly also deleting existing first. Some operations in rclone,
+such as `copy` and `sync` commands, will automatically check for
+`SetModTime` support and re-upload if necessary to keep the modification
+times in sync. Other commands will not work without `SetModTime` support,
+e.g. `touch` command on an existing file will fail, and changes to
+modification time only on a files in a `mount` will be silently ignored.
+
+Storage systems with `R/W`, for read/write, in the ModTime column, means
+they do support modtime-only operations. An `-` means it does not
```

Comment:
This should probably say something like

> A `-` means the modification read on objects is not the modification time of the file when uploaded. It is most likely the time the file was uploaded, or possibly something else (like the time the picture was taken in Google Photos).

Reply:
Nice! I incorporated that description in a new version.